requirements.txt

```diff
@@ -0,0 +1,2 @@
+.
```

Comment:
If we plan to run the server exclusively with Docker, wouldn't it make sense to add this to the setup.py to avoid an additional requirements.txt?

Reply:
the requirements.txt includes gunicorn, which isn't necessary unless you are planning to run a server. For development its a lot better to use the development server included with django, so you don't want to be using either the docker container or gunicorn.

Reply:
Ok makes sense